Fix space below footer

leftclickben opened the pull request:
Remove a JavaScript hack that was causing a large
unwanted white space below the footer.

leftclickben committed:
Fix space below footer

Remove a JavaScript hack that was causing a large
unwanted white space below the footer.

leftclickben commented:
This doesn't work yet, I'll close the PR.  Will reopen after further commits.

leftclickben committed:
Fix page layout issues.

+ Change `#leftNav` and `#mainContent` from absolute positioned to floating.
+ Fix paddings so that float layout works.
+ Add `#leftNavBackground` to remove gap underneath left sidebar.
+ Apply `z-index` to top-level layout components.
+ Remove some spurious javascript.

leftclickben committed:
Fix login page layout issues.

+ Add padding to bottom of body.
+ Override fixed height with `auto` to allow scrolling to work correctly.
+ Recompile the CSS file as minified.

leftclickben commented:
OK this is working correctly now.  The footer is always at the bottom of the page, and the left hand side nav bar always (looks) full height of the page.

kehh commented on themes/default/css/_leftNav.scss:
```diff
@@ -1,18 +1,13 @@
 
 	@include breakpoint(baby-bear) {
-		
+		width: 35%;
```
Should this be on the smallest device

leftclickben replied:
Yeah I think you're right but I think the problem is actually in the definitions of the breakpoints.

leftclickben committed:
Change definition of breakpoints.

+ Specify a `max-width` only for the smallest size
+ Specify a `min-width` only for the largest size
+ Specify both `min-width` and `max-width` for the intermediate sizes

leftclickben committed:
Fix layout issues on small screens.

+ Make the `leftNav` full-width (on small screens)
+ Hide the `leftNavBackground` element (on small screens)
+ Rebuild `local.css`

leftclickben commented:
Have updated based on discussion, including changes to the breakpoint definitions.
